src/keywords/optionalint.cpp

```diff
@@ -3,6 +3,7 @@
 
 #include "keywords/optionalint.h"
 #include "base/lineparser.h"
+#include "base/sysfunc.cpp"
```

Comment:
Eeek! That's not the header...
```suggestion
#include "base/sysfunc.h"
```